packages/office-addin-dev-settings/src/dev-settings-windows.ts

```diff
@@ -17,9 +17,9 @@ const UseDirectDebugger: string = "UseDirectDebugger";
 const UseLiveReload: string = "UseLiveReload";
 const UseProxyDebugger: string = "UseWebDebugger";
 const WebViewSelection: string = "WebViewSelection";
-const IE: string = "ie";
-const Edge: string = "edge";
-const EdgeChromium: string = "edge chromium";
+const WebViewTypeString_IE: string = "ie";
```

Comment:
This is where it would be good to make the display strings mixed case, e.g. "Edge", "IE"